src/components/Board/index.js

```diff
@@ -253,6 +261,8 @@ function BoardContainer({
               }
               disableColumnDrag={disableColumnDrag}
               disableCardDrag={disableCardDrag}
+              onCardAdd={onCardAdd}
```

Comment:
Why do you think we need to call this on a controlled board? I'm not sure about that, the user has full control in his hands on a controlled board. This strategy differs from the addition of a column.